src/fizzBuzz.js

```diff
@@ -0,0 +1,11 @@
+export default function fizzBuzz(numList){
+  for (var i=0; i < numList.length; i++){
+    if ((numList[i] % 15) === 0){
+      numList.splice([i], 1, 'fizzBuzz')
+    } else if ((numList[i] % 5) === 0){
+      numList.splice([i], 1, 'Buzz')
+    } else if ((numList[i] % 3) === 0){
+      numList.splice([i], 1, 'Fizz')
+    } else null
+  } return numList
+}
```

Comment:
This implementation isn't working for me. You have to pass fizzbuzz just the number of times you want it to run. Then update the array based on that number.

```
const fizzBuzz = number => {
  const array = []

  for (let i = 0; i <= number; i++) {
    if (i === 0) {
      array.push(++i)
    } else if (i % 3 === 0 & i % 5 === 0) {
      array.push('FizzBuzz')
    } else if (i % 3 === 0) {
      array.push('Fizz')
    } else if (i % 5 === 0) {
      array.push('Buzz')
    } else {
      array.push(i)
    }
  }

  return array
}
```